.eslintrc.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:ember-suave/recommended',
+  ],
+  env: {
+    qunit: true,
+    node: true,
+  },
+  globals: {
+    // 'expectAssertion': true,
+    // 'expectDeprecation': true,
+    // 'expectNoDeprecation': true,
+    // 'expectWarning': true,
+    // 'expectNoWarning': true,
+    // 'ignoreAssertion': true,
+    // 'ignoreDeprecation': true,
+
+    // Electron
+    'requireNode': true,
+    'processNode': true,
+
+    // A safe subset of "browser:true":
+    // 'window': true,
+    // 'document': true,
+    // 'setTimeout': true,
+    // 'clearTimeout': true,
+    // 'setInterval': true,
+    // 'clearInterval': true,
+
+    // 'Symbol': true,
+    // 'WeakMap': true,
+  },
+  rules: {
+    'comma-dangle': ['error', 'always-multiline'],
+    'ember-suave/no-const-outside-module-scope': 'off',
+    'ember-suave/no-direct-property-access': 'off',
+    'ember-suave/require-access-in-comments': 'off',
+    'newline-before-return': 'error',
+    'new-cap': 'off',
```

Comment:
Had to start somewhere. Turning those on throws quite a few errors in the project.
We can gradually switch them on when the transition is done.

For explantation
- [ember-suave/no-direct-property-access](https://github.com/DockYard/eslint-plugin-ember-suave/blob/master/docs/rules/no-direct-property-access.md) needs a lot of manual adjustments. Should however not be that hard to enable in the long run.
- [ember-suave/require-access-in-comments](https://github.com/DockYard/eslint-plugin-ember-suave/blob/master/docs/rules/require-access-in-comments.md) enforces JSDoc documentation guidelines unused in this project at the time
- [new-cap](http://eslint.org/docs/rules/new-cap) needs very fine tuning as Electron exposes quite a few objects with capital letters which are not used as constructors. Can be done, just a bit much for a quick test.